states/staking.ts

```diff
@@ -0,0 +1,49 @@
+import { atom } from 'jotai'
+
+import { configService } from 'services/config'
+import { uniqAddress } from 'utils/address'
+import { bnum } from 'utils/num'
+import { getTokenInfo } from 'utils/token'
+import { legacyModeAtom } from './userSettings'
+
+export const claimableTokensAtom = atom('0')
```

Comment:
```suggestion
export const claimableTokensAtom = atom('0')
claimableTokensAtom.debugLabel = 'claimableTokensAtom'
```

Added `debugLabel` will show up when used with `jotai/devtools` which you will likely want to use. There's a babel plugin to automate this but I don't think it is worth adding as this will hurt the build performance due to the lack of SWC.